pysoa/server/server.py

```diff
@@ -108,29 +110,50 @@ def handle_next_request(self):
             # Process and run the Job
             job_response = self.process_job(job_request)
 
-            # Send the JobResponse
+            # Prepare the JobResponse for sending by converting it to a message dict
             try:
                 response_message = attr.asdict(job_response, dict_factory=UnicodeKeysDict)
             except Exception as e:
                 self.metrics.counter('server.error.response_conversion_failure').increment()
                 job_response = self.handle_error(e, variables={'job_response': job_response})
                 response_message = attr.asdict(job_response, dict_factory=UnicodeKeysDict)
-            self.transport.send_response_message(request_id, meta, response_message)
 
             response_for_logging = RecursivelyCensoredDictWrapper(response_message)
 
-            if job_response.errors or any(a.errors for a in job_response.actions):
-                if (
-                    self.request_log_error_level > self.request_log_success_level and
-                    self.job_logger.getEffectiveLevel() > self.request_log_success_level
-                ):
-                    # When we originally logged the request, it may have been hidden because the effective logging level
-                    # threshold was greater than the level at which we logged the request. So re-log the request at the
-                    # error level, if set higher.
-                    self.job_logger.log(self.request_log_error_level, 'Job request: %s', request_for_logging)
-                self.job_logger.log(self.request_log_error_level, 'Job response: %s', response_for_logging)
-            else:
-                self.job_logger.log(self.request_log_success_level, 'Job response: %s', response_for_logging)
+            # Send the response message
+            try:
+                self.transport.send_response_message(request_id, meta, response_message)
```

Comment:
How is the response size limit configed? Could we check the size of `response_message` here instead of `try`?

Reply:
We cannot. The response size limit is driven by the transport. (The local transport, for example, doesn't have a limit, but the Redis transport has a 100KB limit.) We have to just wait for the transport to complain.